WordPressKit/WordPressComOAuthClient.swift

```diff
@@ -95,11 +94,12 @@ public final class WordPressComOAuthClient: NSObject {
     private let clientID: String
     private let secret: String
 
-    private let wordPressComBaseUrl: String
-    private let wordPressComApiBaseUrl: String
+    private let wordPressComBaseUrl: URL
+    private let wordPressComApiBaseUrl: URL
 
-    private let oauth2SessionManager: SessionManager = {
-        return WordPressComOAuthClient.sessionManager()
+    // Question: Is it necessary to use these many URLSession instances?
```

Comment:
What do you think of making this a `TODO` or `FIXME` just so tooling can pick it up?

Reply:
Or... have you discovered an answer since writing that comment with the extra work you've done on this library?

Reply:
I didn't use `TODO` or `FIXME` because I'm not sure it needs a change.

But the pattern sure raises a question, at least to me. Because it's pretty rare to see code that uses different URLSession instances to send requests to endpoints that are in the same area (in this case, authentication). But considering the "endpoints" here are all `php` pages, rather than the common REST style API endpoints, maybe it wants to avoid sharing cookies among those API calls?